addons/binding/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/handler/PulseaudioBridgeHandler.java

```diff
@@ -0,0 +1,169 @@
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.smarthome.config.core.Configuration;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.thing.binding.BaseBridgeHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.RefreshType;
+import org.openhab.binding.pulseaudio.PulseaudioBindingConstants;
+import org.openhab.binding.pulseaudio.internal.PulseaudioClient;
+import org.openhab.binding.pulseaudio.internal.items.AbstractAudioDeviceConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class PulseaudioBridgeHandler extends BaseBridgeHandler {
+	private Logger logger = LoggerFactory.getLogger(PulseaudioBridgeHandler.class);
+	
+	public final static Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Collections.singleton(PulseaudioBindingConstants.BRIDGE_THING_TYPE);
+	
+	public String host;
+    public int port;
+    
+    public int refreshInterval;
+    
+    private PulseaudioClient client;
+    
+    private HashSet<String> lastActiveDevices = new HashSet<String>();
+    
+    private ScheduledFuture<?> pollingJob;
+	private Runnable pollingRunnable = new Runnable() {
+		@Override
+		public void run() {
+			client.update();
+			for (AbstractAudioDeviceConfig device : client.getItems()) {
+				if (lastActiveDevices!=null && lastActiveDevices.contains(device.getPaName())) {
+					for (DeviceStatusListener deviceStatusListener : deviceStatusListeners) {
+						try {
+							deviceStatusListener.onDeviceStateChanged(getThing().getUID(), device);
+						} catch (Exception e) {
+							logger.error("An exception occurred while calling the DeviceStatusListener", e);
+						}
+					}
+				}
+				else {
+					for (DeviceStatusListener deviceStatusListener : deviceStatusListeners) {
+						try {
+							deviceStatusListener.onDeviceAdded(getThing(), device);
+							deviceStatusListener.onDeviceStateChanged(getThing().getUID(), device);
+						} catch (Exception e) {
+							logger.error("An exception occurred while calling the DeviceStatusListener", e);
+						}
+						lastActiveDevices.add(device.getPaName());
+					}
+				}
+			}
+		}
+	};
+	
+	private List<DeviceStatusListener> deviceStatusListeners = new CopyOnWriteArrayList<>();
+
+	public PulseaudioBridgeHandler(Bridge bridge) {
+		super(bridge);
+	}
+
+	@Override
+	public void handleCommand(ChannelUID channelUID, Command command) {
+		if (command instanceof RefreshType) {
+			logger.debug("Refresh command received.");
+			client.update();
+			// TODO: postUpdate von neuem State an Item
+			
+		} else
+			logger.warn("No bridge commands defined.");
```

Comment:
put this line in curly brackets - and the message could be changed to something easier understandable, e.g. "received invalid command for bridge xy")